contributors/devel/sig-architecture/conformance-tests.md

```diff
@@ -276,8 +276,7 @@ this approval process:
 - changes to the conformance criteria or process
 - changes to the conformance infrastructure code
 
-There are six columns in this board. New PRs should enter in the To Triage
```

Comment:
I think it makes sense to remove the line since the number of columns keep changing, (at the moment there are 10 columns).